Products/ZenUI3/browser/resources/js/zenoss/AddToZenPackWindow.js

```diff
@@ -31,7 +31,7 @@ Ext.define("Zenoss.AddToZenPackWindow", {
             width: 310,
             plain: true,
             items: [{
-                id: 'addzenpackform',
+                itemId: 'addzenpackform',
```

Comment:
Using "itemId" instead of id to avoid duplication id problems if we create this window multiple times.